scripts/configs.js

```diff
@@ -29,21 +33,22 @@ export const calcEnvironment = () => {
   return environment;
 };
 
-function buildConfigURL(environment) {
+function buildConfigURL(environment, root = '/') {
   const env = environment || calcEnvironment();
   let fileName = 'configs.json';
   if (env !== 'prod') {
     fileName = `configs-${env}.json`;
   }
-  const configURL = new URL(`${window.location.origin}/${fileName}`);
+  const configURL = new URL(`${window.location.origin}${root}${fileName}`);
   return configURL;
 }
 
 const getConfigForEnvironment = async (environment) => {
   const env = environment || calcEnvironment();
+  const root = getRootPath() || '/';
 
   try {
-    const configJSON = window.sessionStorage.getItem(`config:${env}`);
+    const configJSON = window.sessionStorage.getItem(`config:${env}:${root}`);
```

Comment:
We should consider adding retry logic.

That way people on old style syntax config entries in storage will automatically upgrade.

Like one way would be to just immediately clear the config on error (line 53) and refresh the page :)

Reply:
Is this in the scope of multi-store? I don't want to overcomplicate the PR.

Reply:
Well if you are implementing a breaking change, it'll likely cause some discomfort until developers reset session storage. At least it is temporary vs something more persistent.

At the least I would, if I were you, ensure the logs are accurate and helpful for a scenario where user already has `config:prod` or something in their session.

Up to you.

Reply:
We should be fine since the key to the storage is new. We'll test and make sure that's the case.